data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,580 @@ mission "Coalition License Patch"
+
+
+
+mission "Coalition Outbreak 2"
+	name "Collect Vaccines"
+	description "Head to <stopovers>, where Saryd researchers have been working on a vaccine for the new virus, and deliver it to <destination> by <date>."
+	deadline
+	cargo "Vaccines" 80
+	to offer
+		has "Coalition Outbreak 1: done"
+		not "event: coalition virus contained"
+	source "Celestial Third"
+	stopover "Secret Sky"
+	destination "Ki Patek Ka"
+	on offer
+		conversation
+			`As before, the spaceport here is completely dominated by doctors and nurses. You don't find the person who told you to come here, but as you ask one of the other emergency workers here, they direct you to a makeshift office set up here.`
+			`	"Fairly new to us, this virus is," the Kimek manning the stand says. "But finally developed an initial batch of vaccines for it, the researchers on Secret Sky thankfully have."`
+			choice
+				`	"What do you mean "finally"? Didn't this virus just break out recently?"`
+				`	"You want me to go there retrieve the vaccines?"`
+					goto vaccines
+			`	He looks puzzles by your question.`
+			`"That some weeks have already passed, true it is, but hard at work, the researchers have been," he says. "Blame them for taking a few days more than usual, we cannot. Intricate, this virus is."`
+			`	However unaware they may be of it, clearly the Coalition's standards for public health are much more advanced than in other parts of the galaxy you've visited.`
+			label vaccines
+			`	He shows you to a detailed map of Kimek space, explaining that most of the confirmed and suspected cases have been there, as one would expect given that the Kimek population is more than double that of the Arachi and the Saryds combined.`
+			`	"To <destination>, this initial batch must be brought. From there, distributing more batches of the vaccines to other worlds, you will be tasked with."`
+			`	As the line of volunteering merchant captains continues to grow behind you, he is switft to inform you that payment will be <payment>, and asks that the next person come forward.`
+				accept
+	on stopover "Secret Sky"
+		dialog `A group of Saryd workers is waiting for you and they load the <cargo> onto your ship. A laboratory worker babbles about how you must care for them.`
+	on visit
+		dialog `You've reached <planet>, but the vaccines aren't here yet. Depart and wait for whatever ships are carrying them to arrive.`
+	on complete
+		payment 943720
+		"coalition jobs" ++
+		conversation
+			`An incessant stream of freighters has the landing pads of the spaceport here overcrowded, and from the looks of it you only got a quick clearance due to carrying the vaccines.`
+			`	The people here who are surprisingly managing to keep the place organized despite the overwhelming traffic pay you <payment>, as they unload the vaccines off your cargo hold.`
```

Comment:
Sentence flow here feels a bit off, feels like some commas might be needed.

Reply:
is it better now?

Reply:
I was thinking more along the lines of "The people here, who are surprisingly managing to keep the place organized despite the overwhelming traffic, pay you <payment> as they unload the vaccines off your cargo hold."

Reply:
that's... what I did yes. think I can take that as a yes then? :v

Reply:
Sorry, appears that the version I read wasn't the updated one. Ye, it's good now.